buildSrc/src/main/java/com/uber/okbuck/extension/TestExtension.java

```diff
@@ -18,4 +18,7 @@
 
   /** Enable generation of espresso test rules on android libraries. */
   public boolean espressoForLibraries = false;
+
+  /** Enable creation of integration-test rules for modules with `integration-test` folder */
```

Comment:
I think it will default to `integrationTest` now? please also update the comment.